config/application.rb

```diff
@@ -33,5 +33,7 @@ class Application < Rails::Application
 
     ActiveModelSerializers.config.adapter = :json_api
     FileUtils.mkdir_p("pages")
+
+    config.active_job.queue_adapter = :sidekiq
```

Comment:
This is not necessary if you're not using ActiveJob, which, due to `include Sidekiq::Job`, you are not.

Reply:
done